lib/str.js

```diff
@@ -1049,6 +1050,18 @@ const Str = {
     isDomainEmail(email) {
         return this.startsWith(email, '+@');
     },
+
+    /**
+     * Polyfill for String.prototype.replaceAll
+     *
+     * @param {String} text
+     * @param {String|RegExp} searchValue
+     * @param {String|Function} replaceValue
+     * @returns {String}
+     */
+    replaceAll(text, searchValue, replaceValue) {
```

Comment:
Just kind of curious why we have to import a package for this, why can't we use inbuilt [replaceAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll)?

Reply:
Mostly just to backfill support for browsers where it's not available

https://caniuse.com/?search=replaceAll

Reply:
We could implement a polyfill in every repository, but I think it is just easier in this case to add it to our "string utility library" which several other repos are using.

Reply:
Thanks for clearing this out I can see ios Safari `3.2 - 13.3` is not supported. 🙌